src/content/plugins/source-map-dev-tool-plugin.md

```diff
@@ -45,3 +47,14 @@ new webpack.SourceMapDevToolPlugin({
   exclude: ['vendor.js']
 })
 ```
+
+### Host sourcemaps externally
```

Comment:
I think we should captilize the title, split "sourcemaps" into "Source Maps", and maybe simplify slightly to "External Source Maps" (though I'm also ok with "Host Source Maps Externally").